tests/Responses/Expected/shipment/shipment_index_response.json

```diff
@@ -0,0 +1,191 @@
+                    "lastName": "Mukhin",
+                    "countryCode": "UA",
+                    "street": "Main street",
+                    "city": "Odesa",
+                    "postcode": "65000",
+                    "phoneNumber": "380951234567"
+                },
+                "adjustments": {
+                    "UPS": {
+                        "amount": 500,
+                        "neutral": false
+                    },
+                    "TAX_EU": {
+                        "amount": 399,
+                        "neutral": true
+                    },
+                    "PROMO_UNIT_X": {
+                        "amount": -198,
+                        "neutral": false
+                    }
+                },
+                "itemsTotal": 3800,
+                "total": 4300,
+                "adjustmentsTotal": 302,
+                "customer": {
+                    "id": @integer@,
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "email": "igor.mukhin@gmail.com",
+                    "birthday": "@string@.isDateTime()",
+                    "gender": "m",
+                    "phoneNumber": "380951234567"
+                }
+            },
+            "units": [
+                {
+                    "shippable": {
+                        "id": @integer@,
+                        "code": "red-mug",
+                        "name": "Red Mug",
+                        "barcode": "7891000074237",
+                        "weight": 125,
+                        "onHand": 7,
+                        "images": [
+                            {
+                                "type": "thumbnail",
+                                "path": "/pants.jpeg",
+                                "cachedPath": "http://localhost/media/cache/setono_sylius_lagersystem/pants.jpeg"
+                            }
+                        ]
+                    },
+                    "total": 1900,
+                    "adjustmentsTotal": -99
+                },
+                {
+                    "shippable": {
+                        "id": @integer@,
+                        "code": "red-mug",
+                        "name": "Red Mug",
+                        "barcode": "7891000074237",
+                        "weight": 125,
+                        "onHand": 7,
+                        "images": [
+                            {
+                                "type": "thumbnail",
+                                "path": "/pants.jpeg",
+                                "cachedPath": "http://localhost/media/cache/setono_sylius_lagersystem/pants.jpeg"
+                            }
+                        ]
+                    },
+                    "total": 1900,
+                    "adjustmentsTotal": -99
+                 }
+            ]
+        },
+        {
+            "id": @integer@,
+            "state": "partially_shipped",
```

Comment:
Can a shipment be partially shipped?

Reply:
You're right, single shipment can be only `shipped`, `cancelled`, `ready` and `cart`. 
`partially_shipped` is about order's `shippingState`...

Reply:
This one was fixed. And added two shipments to one order just to see how response looks like in that case.